app/views/layouts/_tracking_scripts_preconnect.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if Rails.env.production? %>
+  <link rel="preconnect" href="https://script.hotjar.com" crossorigin>
+  <link rel="preconnect" href="https://vars.hotjar.com" crossorigin>
+  <link rel="preconnect" href="https://bid.g.doubleclick.net" crossorigin>
+  <link rel="preconnect" href="https://www.google-analytics.com" crossorigin>
+  <link rel="preconnect" href="https://googleads.g.doubleclick.net" crossorigin>
+  <link rel="preconnect" href="https://cdnjs.cloudflare.com" crossorigin>
```

Comment:
Can we preconnect Google Fonts as well?

Reply:
From my understanding, since we're directly linking to google fonts (and not subsequently linking in a stylesheet, for example), we don't need to preconnect.